transaction.go

```diff
@@ -461,7 +462,7 @@ func (txn *Txn) batchPrewriteSecondaryRowsWithLockClean(tbl []byte, rowMs map[st
 				return errors.Trace(err)
 			}
 			if lock != nil {
-				return fmt.Errorf("can't clean lock, column:%+v; conflict lock: %+v, lock ts: %d", lock.getColumn(), lock, lock.getTimestamp())
```

Comment:
can't use %v for lock.GetColumn, it returns binary data,and console will make a "peng" noise .